test/storage/cookie_storage_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class CookieStorageTest < Minitest::Test
+  def setup
+    @storage = storage = Verdict::Storage::CookieStorage.new
+    @experiment = Verdict::Experiment.new(:cookie_storage_test) do
+      name 1234
+      groups do
+        group :all, 100 do name 5678
+        end
+      end
+      storage storage, store_unqualified: true
+    end
+    @subject = stub(id: 'bob')
+    @assignment = Verdict::Assignment.new(@experiment, @subject, @experiment.group(:all), nil)
+  end
+
+  def test_cookie_lifespan_has_a_default
+    refute_nil Verdict::Storage::CookieStorage.new.cookie_lifespan
+  end
+
+  def test_cookie_lifespan_can_be_configured
+    storage = Verdict::Storage::CookieStorage.new(cookie_lifespan: 60)
+
+    assert_equal 60, storage.cookie_lifespan
+  end
+
+  def test_store_assignment_returns_true_when_an_assignment_is_stored
+    assert @storage.store_assignment(@assignment)
```

Comment:
This should be asserting the assignment is actually stored.

Reply:
I um'd and ah'd about this and the reason this just tests boolean is because that's all the interface suggests, "Returns true when successful". 

The next test stores an assignment and then retrieves it `test_retrieve_assignment_returns_an_assignment`, perhaps I could combine the two?

Reply:
I think this is a symptom of the choice of allowing a hash via defaults. I think we shouldn't supply a default value for cookie storage because the zero arg constructor fundamentally won't work the way someone might think it would. If we pass in a hash we explicitly have reference to it in tests to assert on.

Reply:
Similar to the previous comment, the tests now have a cookie jar instance so I've added an additional assertion that after storing, the retrieval is not nil.